app/page-partials/admin-dashboard/AdminTabs/BceidTabContent.tsx

```diff
@@ -10,7 +10,7 @@ function BceidTabContent({ integration, onApproved }: Props) {
   if (!integration) return null;
   const { status, bceidApproved } = integration;
 
-  const canApproveProd = (status === 'applied' && !bceidApproved) || false;
+  const canApproveProd = !bceidApproved || false;
```

Comment:
not sure if we want to approve things when the integration is in a progress.

Reply:
@junminahn the button will be disabled anyway when TF is in  progress -> https://github.com/bcgov/sso-requests/blob/2093c3b1c2c8997b85408b3fa6217b70da097c6b/app/page-partials/admin-dashboard/AdminTabs/TabContent.tsx#L39